red/api/info.js

```diff
@@ -43,6 +43,10 @@ module.exports = {
             safeSettings.flowFilePretty = settings.flowFilePretty;
         }
 
+        if (settings.tlsDisableLocalFiles) {
```

Comment:
I've added [this](https://trello.com/c/jmFK6hAb/87-allow-a-node-to-declare-what-settings-should-be-made-available-to-the-editor) to trello - I think we need a mechanism for a node to register what settings it needs the runtime to expose to the editor. That way any node can do this without depending on us updating this hardcoded list.

(This is an FYI - no changes need in the PR, but we will update it later)